src/settings/command_line.rs

```diff
@@ -24,10 +26,13 @@ pub struct Opt {
     /// $XDG_CONFIG_HOME/qldbshell/config.ion
     #[structopt(short, long, parse(from_os_str))]
     pub config: Option<PathBuf>,
-
+    /// The `qldb-session` API endpoint to connect to.
+    /// For a complete list of available QLDB Regions and endpoints,
+    /// see [Amazon QLDB endpoints and quotas](https://docs.aws.amazon.com/general/latest/gr/qldb.html).
     #[structopt(short = "-s", long = "--qldb-session-endpoint", parse(try_from_str = Url::try_from))]
     pub qldb_session_endpoint: Option<Url>,
-
+    /// The location of your AWS credentials profile to use for authentication.
+    /// If not provided, the shell uses your default AWS profile, which is located at `~/.aws/credentials`.
```

Comment:
... and can be overridden by an environment variable!

Similar to the other comment, I think we should link to accurate and complete documentation rather than potentially misleading users by having incorrect information.

Reply:
Updated